facebook-commerce-events-tracker.php

```diff
@@ -559,6 +582,19 @@ public function inject_initiate_checkout_event() {
 				],
 			];
 
+			// if there is only one item in the cart, send its first category
+			if ( ( $cart = WC()->cart ) && count( $cart->get_cart() ) === 1 ) {
```

Comment:
What if there are more items, but only one category? And not set on the very first item?

Reply:
Handling multiple items with the same category will probably have to be out of scope for now given the timeframe. I don't think a product can have no category, but I added a check just in case f56841b185f682080ce551dc4756c6b0c7b52818